File: src/user-service/src/main/java/org/palette/easeluserservice/api/dto/request/JoinRequest.java
package org.palette.easeluserservice.api.dto.request;

import java.util.Optional;

public record JoinRequest(
        String email,
        String password,
        String username, // @DoHyeon
        String nickname, // DoHyeon
        Optional<String> introduce,
        Optional<String> profilePath,
        Optional<String> backgroundPath,
        Optional<String> websitePath
) {
}

File: src/mobile/app/src/main/java/com/smilegate/Easel/domain/model/join/JoinRequest.kt
package com.smilegate.Easel.domain.model.join

data class JoinRequest(
    val email: String,
    val password: String,
    val username: String,
    val nickname: String,
    val introduce: String? = null,
    val profileImagePath: String? = null,
    val websitePath: String? = null
)
